sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkTest.java

```diff
@@ -539,16 +664,49 @@ void configurationError_ClosesResources() {
                     .addLoggerProviderCustomizer((u1, u2) -> loggerProviderBuilder)
                     .addPropertiesSupplier(() -> singletonMap("otel.metrics.exporter", "none"))
                     .addPropertiesSupplier(() -> singletonMap("otel.traces.exporter", "none"))
-                    .addPropertiesSupplier(() -> singletonMap("otel.logs.exporter", "none"))
-                    .addPropertiesSupplier(() -> singletonMap("otel.propagators", "foo"))
+                    .addPropertiesSupplier(() -> singletonMap("otel.logs.exporter", "foo"))
+                    .addPropertiesSupplier(() -> singletonMap("otel.sdk.disabled", "false"))
                     .build())
         .isInstanceOf(ConfigurationException.class)
-        .hasMessageContaining("Unrecognized value for otel.propagators");
+        .hasMessageContaining("Unrecognized value for otel.logs.exporter: foo");
 
     verify(tracerProvider).close();
     verify(meterProvider).close();
-    verify(loggerProvider).close();
 
     logs.assertContains("Error closing io.opentelemetry.sdk.trace.SdkTracerProvider: Error!");
   }
+
+  @Test
+  @SuppressLogger(AutoConfiguredOpenTelemetrySdkBuilder.class)
+  void configurationError_fileNotFound() {
+    assertThatThrownBy(
+            () ->
+                AutoConfiguredOpenTelemetrySdk.builder()
+                    .addPropertiesSupplier(() -> singletonMap("otel.config.file", "foo"))
+                    .addPropertiesSupplier(
+                        () -> singletonMap("otel.experimental.config.file", "foo"))
+                    .addPropertiesSupplier(() -> singletonMap("otel.sdk.disabled", "true"))
+                    .build())
+        .isInstanceOf(ConfigurationException.class)
+        .hasMessageContaining("Configuration file not found");
+
+    Assertions.assertDoesNotThrow(
+        () ->
+            AutoConfiguredOpenTelemetrySdk.builder()
+                .addPropertiesSupplier(() -> singletonMap("otel.experimental.config.file", ""))
+                .addPropertiesSupplier(() -> singletonMap("otel.sdk.disabled", "true"))
+                .build());
+  }
+
+  @Test
+  @SuppressLogger(AutoConfiguredOpenTelemetrySdkBuilder.class)
+  void serviceClassLoader() {
+    ClassLoader classLoader = null;
+    assertThatThrownBy(
+            () -> AutoConfiguredOpenTelemetrySdk.builder().setServiceClassLoader(classLoader))
+        .isInstanceOf(NullPointerException.class)
+        .hasMessageContaining("serviceClassLoader");
+    ClassLoader mockClassLoader = mock(ClassLoader.class);
```

Comment:
I think this is what's causing the test to fail: https://github.com/mockito/mockito/issues/1696#issuecomment-488052223

I can recreate the test failure locally on my machine.